examples/vlan_learning.c

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) PLUMgrid, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License")
+
+#include <bcc/proto.h>
+
+struct ifindex_leaf_t {
+  int out_ifindex;
```

Comment:
Not sure I understand. Are you suggesting to pack the structure, or put some pad fields in there, or something else?